Tests/FreddyTests/JSONSerializingTests.swift

```diff
@@ -17,18 +16,9 @@ class JSONSerializingTests: XCTestCase {
         XCTAssertFalse(string.isEmpty, "There should be characters.")
     }
 
-    func testThatJSONDataIsEqual() {
-        let serializedJSONData = try! json.serialize()
-        let noWhiteSpaceJSON = try! JSON(data: noWhiteSpaceData)
-        let noWhiteSpaceSerializedJSONData = try! noWhiteSpaceJSON.serialize()
-        XCTAssertEqual(serializedJSONData, noWhiteSpaceSerializedJSONData, "Serialized data should be equal.")
-    }
-    
-    func testThatJSONStringIsEqual() {
```

Comment:
Can you open this one up some more, please?
I don't get how these test removals map to the PR description.

Reply:
(It looks to me like there is one test less there.)

Reply:
Clarification above https://github.com/mpapp/Freddy/pull/1#discussion_r276239661